tools/cldr-code/src/main/resources/org/unicode/cldr/util/downgrade.txt

```diff
@@ -5,8 +5,10 @@
 # If you need to match to the end, follow with $
```

Comment:
If this file were in util/data rather than in util, then maybe methods like CldrUtility.getUTF8Data would be easier to use -- is there a reason it's not in util/data?